docs/android/README.md

```diff
@@ -550,6 +525,15 @@ You can find the API documentation in [PSPDFKit.js](../../www/PSPDFKit.js).
 
 ## Troubleshooting
 
+### PSPDFKit runs in trial mode
+
+If you're already a customer then please make sure that the package ID matches with your bundle ID that's assigned to your license (e.g. io.ionic.starter). You can check this in your `AndroidManifest.xml` by searching for `package`.
+
+Make sure that you are setting the license key via `PSPDFKit.setLicenseKey()`.
+If you used the PSPDFKit Cordova plugin previously, you might still have `pspdfkit.license=YOUR LICENSE` set in `local.properties`. 
+This is no longer supported.
+Please make sure to migrate this to use `PSPDFKit.setLicenseKey("YOUR LICENSE")`.
```

Comment:
Should we mention that they should not call this method if they want to run it in trial mode? Right now, this is not explicitly stated, so a reader might wonder what to do.

Reply:
Yes, that makes sense. I'll mention that. Thank you!